src/js/controller/setup-steps.js

```diff
@@ -126,7 +126,9 @@ define([
         if (_.isString(playlist)) {
             _playlistLoader = new PlaylistLoader();
             _playlistLoader.on(events.JWPLAYER_PLAYLIST_LOADED, function(data) {
-                _model.set('playlist', data.playlist);
+                for (var key in data) {
+                    _model.set(key, data[key]);
+                }
```

Comment:
Too risky. As a work-around. Copy data.feedid is present. We don't want any other fields from this data.